src/BenchmarksDriver/Program.cs

```diff
@@ -444,6 +454,7 @@ public static int Main(string[] args)
                         [ConnectionFilter] [nvarchar](max) NULL,
                         [WebHost] [nvarchar](max) NOT NULL,
                         [KestrelThreadCount] [int] NULL,
+                        [KestrelThreadPoolDispatching] [bit] NULL,
```

Comment:
Surfacing comment that's been buried&hellip;

Thought we were going to change [KestrelThreadCount] to NOT NULL after updating the main database?

Either way, might be good to do that for this new column. Would store true when not overridden on the command line.